Enterprise-CMCS/macpro-mdct-qmr
# [QMR] 508 Remediation - Labels for Sensory Characters

### Description

Added tooltip for icons in QMR

### Related ticket(s)


MDCT-1949

---
### How to test

- Login
- - Confirm new AX functionality: 
    - Three icons were affected: More Actions, Delete, and Date Picker -- all should have a tooltip when hovering 
    - When tabbing through months in the Date Picker, voiceover should now read the year along with the month. e.g.:"May 2022, June 2022, etc"
- More actions icon can be found on the main page
- Delete icon can be found under adding another category under OMS
- Date Picker is found by clicking "No'' on a date range question


### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
